tests/sentry/web/frontend/test_auth_login.py

```diff
@@ -176,6 +176,20 @@ def test_login_valid_credentials_with_org_and_customer_domains(self):
             (f"http://{org.slug}.testserver/issues/", 302),
         ]
 
+    @with_feature("system:multi-region")
+    def test_redirect_to_login_with_org_and_customer_domains(self):
+        org = self.create_organization(owner=self.user)
+        self.create_project(organization=org, name="project")
+        # load it once for test cookie
+        self.client.get(self.path)
+
+        project_path = reverse("project-details", kwargs={"project_slug": "project"})
+        resp = self.client.get(project_path, HTTP_HOST=f"{org.slug}.testserver")
+
+        assert resp.status_code == 302
+        # redirect to auth org login page by parsing customer domain
+        assert getattr(resp, "url", None) == reverse("sentry-auth-organization", args=[org.slug])
```

Comment:
can we also load the url here and ensure that we get a 200 request after redirecting? (want to ensure we don't end up in a redirect loop)

Reply:
done, i actually checked and the logic before the fix would lead to a 302 :O